### PR TITLE
Add pyraf def

### PR DESCRIPTION
**Pull request by Tanvez**
This PR is mainly to add the `pyraf.def` with the remote desktop app template from this [repo](https://github.com/Harvard-ATG/ood-remote-desktop-rosgazebo).
Note: This iteration uses Python 2.7 due to what packages are available in the conda package library. To use iraf/pyraf you will need to run `conda init` and `conda activate iraf`

### CODE REVIEW

File: template/before.sh.erb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+XDG_MUSIC_DIR="$HOME/Music"
+XDG_PICTURES_DIR="$HOME/Pictures"
+XDG_PUBLICSHARE_DIR="$HOME/Public"
+XDG_TEMPLATES_DIR="$HOME/Templates"
+XDG_VIDEOS_DIR="$HOME/Videos"
+EOF
+echo "TIMING: $(date -Iseconds) - set up user dirs folder"
+
+# Create a desktop icon for Gazebo
+cat > $DESKTOP_FOLDER/Gazebo.desktop <<EOF
```
Comment (arthurian) [inline template/before.sh.erb]: Is the Gazebo desktop icon supposed to work for pyraf? There's also a VSCodium and Matlab desktop icon.

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+## Installation
+In order to install the app
+
+Note you will need to be in the head node while building any of the apptainer images.
+Apptainer images should be located `/shared/apptainerImages`
+
+- Activate the spack environment and then use activate the apptainer env.
+```bash
+root@ip:/# . shared/spack/share/spack/setup-env.sh
+root@ip:/# spacktivate apptainer
```
Comment (arthurian) [inline README.md:13]: Neat, didn't know there was an alias for `spack env activate`.

File: README.md [hunk 3]
```diff
@@ -1 +1,18 @@
...
+```
+- Build the image from the `pyraf.def` file by running `apptainer build pyraf.sif pyraf.def` Note: the `pyraf.sif` should be available in the `/shared/apptainerImages` folder
+
+- After the image is successful you can add the app by following the [Adding OOD app runbook](https://docs.google.com/document/d/1LejSmmgY7rJDBwl1gl-Q835lSbf6v2b2J3TFhxPNwfc/edit?tab=t.0#heading=h.n0uvlxdrj7e3)
```
Comment (arthurian) [inline README.md]: Just a note that this github repo is public, but the linked google doc is not. I don't think this is a problem, but we should be aware when publishing public repos.

**Review by arthurian — APPROVED** [review]
Overall looks good @Tanvez! Great job getting this migrated to OOD. 👍

File: README.md (hunk 2 above)
Comment (jaguillette) [reply README.md:13]: There's a corresponding `despacktivate` too

File: README.md (hunk 3 above)
Comment (jaguillette) [reply README.md]: If we do publish public repos with links to non-public documents (which I also think is fine), is it worth calling out with something like `(internal HUIT documentation)` after it? I think we could also just wait for it to come up, possibly with a random access request or GH issue.

File: build/pyraf.def [hunk 4]
```diff
@@ -0,0 +1,65 @@
+Bootstrap: docker
+From: ubuntu:20.04
+
+%post
+# Install packages needed for baseline desktop and ros installation
```
Comment (jaguillette) [inline build/pyraf.def]: Nit: This line has another trailing `ros` reference

File: template/before.sh.erb (hunk 1 above)
Comment (jaguillette) [reply template/before.sh.erb]: Codium wouldn't be bad to have, but I don't see it getting installed in the container definition. Gazebo and Matlab aren't getting installed either, as far as I can see.

**Review by jaguillette — APPROVED** [review]
Same here, looks good to me! The one thing I'd try to change before deploying is to remove the extra desktop icons. Looks great though!

File: template/before.sh.erb (hunk 1 above)
Comment (Tanvez) [reply template/before.sh.erb]: Must have missed this. Will remove as we dont have a use for Gazebo, matlab, or VScodium

File: README.md (hunk 3 above)
Comment (Tanvez) [reply README.md]: I like the idea of adding a disclaimer for now and if folks are busting down our door for the runbook we can add it to the readme.

**Pull request merged**